exec/jdbc/pom.xml

```diff
@@ -134,6 +134,7 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-common</artifactId>
+          <scope>test</scope>
```

Comment:
I don't think this is a correct change. Hadoop-common is actually a dependency of the jdbc package. I think your issue with a larger jdbc-all is due to a buggy version of maven.